### PR TITLE
PORTALS-3101

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: apps/SageAccountWeb/src/components/ConfigureEmail.tsx [hunk 1]
```diff
@@ -69,6 +69,7 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
           accessToken,
         ).then(() => {
           displayToast('Email has been successfully added', 'success')
+          refetchCurrentProfile()
```
Comment (jay-hodgson) [inline apps/SageAccountWeb/src/components/ConfigureEmail.tsx:72]: PORTALS-3101: Refetch after new email has been successfully added to the account

File: apps/SageAccountWeb/src/components/ConfigureEmail.tsx [hunk 2]
```diff
@@ -99,6 +100,7 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
     try {
       await SynapseClient.deleteEmail(accessToken, email)
       refetchCurrentProfile()
+      displayToast('Email has been successfully removed', 'success')
```
Comment (jay-hodgson) [inline apps/SageAccountWeb/src/components/ConfigureEmail.tsx:103]: Noticed that there was no notification when an email was removed

File: packages/synapse-react-client/src/synapse-client/SynapseClient.ts [hunk 3]
```diff
@@ -4256,7 +4256,7 @@ export const addEmailAddressStep2 = (
  */
 export const deleteEmail = (accessToken: string | undefined, email: string) => {
   return doDelete(
-    `/repo/v1/email?email=${email}`,
+    `/repo/v1/email?email=${encodeURIComponent(email)}`,
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/synapse-client/SynapseClient.ts:4259]: And an email with a '+' was not encoded correctly in this call

File: apps/SageAccountWeb/src/components/ConfigureEmail.tsx (hunk 2 above)
Comment (hallieswan) [reply apps/SageAccountWeb/src/components/ConfigureEmail.tsx:103]: Could consider adding a notification to `changePrimaryEmail` when the primary email was successfully changed, but up to you!

**Review by hallieswan — APPROVED** [review]
Looks good to me!

File: apps/SageAccountWeb/src/components/ConfigureEmail.tsx (hunk 2 above)
Comment (jay-hodgson) [reply apps/SageAccountWeb/src/components/ConfigureEmail.tsx:103]: Good idea

**Review by hallieswan — APPROVED** [review]

**Pull request merged**